styles.css

```diff
@@ -0,0 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@900&display=swap');
+
+main {
+  display: flex;
+  flex-direction: column;
+  max-width: 1000px;
+  margin: 5%;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  color: #6070ff;
+  position: relative;
+  display: flex;
+  align-items: baseline;
+}
+
+h4 {
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  position: static;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+h4:hover {
+  cursor: pointer;
+}
+
+.hamb {
+  width: 18px;
+  float: right;
+  top: 18px;
+  right: 4px;
+  position: absolute;
+  color: #6070ff;
+}
+
+.hamb:hover { cursor: pointer; }
+
+section {
+  text-align: center;
+}
```

Comment:
Good job on following the [figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) but please add background as it is on design. Here you have a screenshot of the design how it should be: 

![image](https://user-images.githubusercontent.com/74506933/153273946-403dcc22-ba98-45a1-b68a-7047a6fd6e40.png)